portal/portal-impl/impl/src/bundle/sitenav.properties

```diff
@@ -71,7 +71,7 @@ sit_othersitesclose=Close this drawer
 sit_refresh_favorites=<a href=\"javascript:location.reload()\">Reload</a> to see your updated favorite sites
 sit_star_sites_description=Star sites below to add them to your favorites bar.
 sit_star_sites_with_autofav_description=Star sites below to add them to your favorites bar. New sites will be automatically added to your favorites.
-sit_drag_sites_to_reorder=Drag sites to reorder them. Up to {0} sites can be displayed in your favorites bar.
+sit_drag_sites_to_reorder=Click and drag the <i class=\"fa fa-bars\"></i> icon to reorder your favorites. Up to {0} sites can be displayed in your favorites bar.
```

Comment:
is this i class being rendered as an icon? If yes, you should add aria-hidden to avoid screen reader reading it

Reply:
Yes, it's rendered as an icon. I'll make than change, thanks!

Reply:
All fixed up, thanks @ottenhoff